packages/blender.rb

```diff
@@ -19,7 +19,7 @@ class Blender < Package
   def self.install
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/blender"
-    FileUtils.mv Dir.glob('*'), "#{CREW_DEST_PREFIX}/share/blender"
+    FileUtils.mv Dir['*'], "#{CREW_DEST_PREFIX}/share/blender"
     FileUtils.ln_s "#{CREW_PREFIX}/share/blender/blender", "#{CREW_DEST_PREFIX}/bin/blender"
```

Comment:
Use relative symlink?
```
FileUtils.ln_s '../share/blender/blender', "#{CREW_DEST_PREFIX}/bin/blender"
```

Reply:
The absolute symlink works the same and is easier to understand.